From 307a43cf501f6b0b9fcb258a6753afcaf05e59c7 Mon Sep 17 00:00:00 2001
From: Ales Kozumplik <ales@redhat.com>
Date: Mon, 23 Jul 2012 10:44:24 +0200
Subject: [PATCH] solv.i: new perl includes <stdbool.h> and the custom
 typedefs cause gcc errors.

Use <stdbool.h> as the default instead.
---
 bindings/solv.i |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/bindings/solv.i b/bindings/solv.i
index a5b9552..618401c 100644
--- a/bindings/solv.i
+++ b/bindings/solv.i
@@ -331,6 +331,7 @@ typedef VALUE AppObjectPtr;
 %include "typemaps.i"
 
 %{
+#include <stdbool.h>
 #include <stdio.h>
 #include <sys/stat.h>
 #include <sys/types.h>
@@ -378,9 +379,6 @@ typedef VALUE AppObjectPtr;
 #define RARRAY_LEN(ary) (RARRAY(ary)->len)
 #endif
 
-#define true 1
-#define false 1
-
 #define SOLVER_SOLUTION_ERASE                   -100
 #define SOLVER_SOLUTION_REPLACE                 -101
 #define SOLVER_SOLUTION_REPLACE_DOWNGRADE       -102
@@ -388,7 +386,6 @@ typedef VALUE AppObjectPtr;
 #define SOLVER_SOLUTION_REPLACE_VENDORCHANGE    -104
 
 typedef struct chksum Chksum;
-typedef int bool;
 typedef void *AppObjectPtr;
 
 typedef struct {
-- 
1.7.10.2

